test/language/directive-prologue/func-decl-final-runtime.js

```diff
@@ -2,15 +2,16 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
+esid: use-strict-directive
 es5id: 10.1.1-21-s
 description: >
     Strict Mode - Function code of a FunctionDeclaration contains Use
     Strict Directive which appears at the end of the block
 flags: [noStrict]
 ---*/
         function fun() {
-            eval("var public = 1;");
-            assert.sameValue(public, 1);
+            test262unresolvable = null;
+            assert.sameValue(test262unresolvable, null);
             "use strict";
```

Comment:
I wonder if we have some coverage for the pragma only preceded by an empty statement. I did not find anything from a quick grep.

```js
function fun() {
  ;'use strict';
  unr = null;
  assert.sameValue(unr, null);
}
fun();
```